### PR TITLE
approved values of maker and user

### PR DESCRIPTION
**Pull request by Danwhy**
ref #160, #418 
Suppresses role of maker/user/relation if not an approved value

### CODE REVIEW

File: lib/transforms/get-values.js [hunk 1]
```diff
@@ -140,6 +141,9 @@ module.exports = {
           key = getNestedProperty(el, 'attributes.role.value') || getNestedProperty(el, 'attributes.role.type');
           value = getNestedProperty(el, 'attributes.summary_title') || null;
           link = getNestedProperty(el, 'links.self');
+          if (makersList.indexOf(key) === -1) {
+            key = 'maker';
```
Comment (SimonLab) [inline lib/transforms/get-values.js]: Just to clarify, if the type of the maker is not in the list we are defining the type "maker" by default, is that right?

Comment (Danwhy) [inline lib/transforms/get-values.js]: Yeah that's right

**Comment by nelsonic** [comment]
pesky merge conflicts! (_please re-assign when resolved, thanks!_)

**Comment by nelsonic** [comment]
@Danwhy merge conflicts ... 😞  (_please resolve and re-assign_)
(_code looks good tho..._)

**Comment by nelsonic** [comment]
@Danwhy thanks for fixing. 👍 
(_@SimonLab thanks for doing initial review ... code great as always!_)

**Pull request merged**